src/lib/djerba/configure.py

```diff
@@ -178,6 +155,31 @@ def discover_secondary(self):
         updates[ini.SEQUENZA_SOLUTION] = solution
         updates[ini.PLOIDY] = ploidy
         updates[ini.PURITY] = purity
+        tumour_id =  self.config[ini.DISCOVERED][ini.TUMOUR_ID]
+        try:
+            coverage = pull_qc(self.config).fetch_coverage_etl_data(tumour_id)
+        except MissingQCETLError as e:
+            msg = "Error {0}. Djerba couldn't find the coverage associated with tumour_id {1} in QC-ETL. Defaulting target coverage to .ini parameter.".format(e.code,tumour_id)
```

Comment:
`MissingQCETLError` probably doesn't have a `code` attribute, that's specific to `HTTPError`.

Instead I'd do something like:
`msg = "Can't retrieve coverage associated with tumour_id {0} in QC-ETL. Will use manually configured INI parameter, if available: {1}".format(tumour_id, e)`

This is clearer, and calling `format` on an Exception object will give you a reasonably helpful printout of its details.

BTW it's not necessary to say "error" or mention Djerba in your message, you get both of those for free from the logger, eg:
`
2023-03-28_09:59:52 djerba.util.validator ERROR: [discovered] field 'sample_name_whole_genome_normal' cannot be an empty string
`